### DISCUSSION TITLE
Airbyte Sync Failing

### DISCUSSION THREAD

**Discussion by nazifishrak** [airbytehq/airbyte#28352] [Connector Questions]
Hello I was trying to do an incremental sync using airbyte with the following specs:
Deployment: Docker on VM 
Airbyte version: 0.44
**Source: Zendesk
Destination: BQ**
Step: during sync
OS: Debian 11
Memory: 15 GB
Processor: 4 vCPU

We had data pulled till 13th July and yesterday we tried doing an incremental sync and we had faced errors while syncing.
I have attached the log files for the 2 attempts we did.
Any help or guidance is appreciated
[attemp1.txt](https://github.com/airbytehq/airbyte/files/12073086/attemp1.txt)

[attempt2.txt](https://github.com/airbytehq/airbyte/files/12073088/attempt2.txt)

**Comment by marcosmarxm** [comment]
Can you update the log file, kind difficult to read the logs you shared.

**Reply by nazifishrak** [reply]
[attemp1.txt](https://github.com/airbytehq/airbyte/files/12073068/attemp1.txt)
[attempt2.txt](https://github.com/airbytehq/airbyte/files/12073069/attempt2.txt)

**Reply by marcosmarxm** [reply]
From the logs, it looks the source threw an error. Can you update the description of your problem adding more information about:
* what is the connector version for both source and destination.

```
2023-07-16 08:42:23 INFO i.a.w.g.BufferedReplicationWorker(readFromSource):316 - readFromSource: exception caught
java.io.UncheckedIOException: java.io.IOException: Stream closed
	at java.io.BufferedReader$1.hasNext(BufferedReader.java:685) ~[?:?]
```

Are you able to create a new connection with another source and have successful sync (maybe use Faker or Poke API)?

**Reply by nazifishrak** [reply]
The connector version is as follows:

```
LABEL io.airbyte.version=0.10.0
LABEL io.airbyte.name=airbyte/source-zendesk-support

LABEL io.airbyte.version=1.5.1
LABEL io.airbyte.name=airbyte/destination-bigquery
```
We also have other syncs with MongoDB, and Postgresql Source and the sync works successfully.

**Reply by nazifishrak** [reply]
Also upon checking our GCP, we can see the data has synced. The Zendesk support <--> BQ connection failed from the logs. However, the sync to BQ was a success according to BQ. How is it possible?

**Reply by marcosmarxm** [reply]
My recommendation is: 
* make sure the source is updated with latest version
* check if all parameters in your source are updated and correct
* try to recreate the connection
* try to use another destination
